doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -53,10 +53,32 @@ REST API Version History: https://docs.openstack.org/cinder/latest/contributor/a
 
 ### 5.2.4. Swift
 
-| **OpenStack Service** | **Link for API and CLI** | **API/CLI Minimum (Baseline) Version** |
-|------------------|----------------------------------------------------|-------------------|
-| Object Storage: Swift | https://docs.openstack.org/api-ref/object-store/ | Version 1.0 |
-| Object Storage: Swift |https://docs.openstack.org/python-swiftclient/latest/ | Version 1.0 |
+| **OpenStack Service** | **Version** |
```

Comment:
Can you rename Version -> API Version

Reply:
Done. It's part of the API section but if it helps.